litho-testing/src/main/java/com/facebook/litho/testing/TestDrawableComponent.java

```diff
@@ -224,17 +197,23 @@ public static Builder create(
       boolean implementsAccessibility,
       boolean usesDisplayList,
       boolean isMountSizeDependent) {
-    return newBuilder(
-        context,
-        defStyleAttr,
-        defStyleRes,
-        new State(
-            callsShouldUpdateOnMount,
-            isPureRender,
-            canMeasure,
-            implementsAccessibility,
-            usesDisplayList,
-            isMountSizeDependent));
+
+    byte properties = 0;
+    
+    if (callsShouldUpdateOnMount) 
```

Comment:
Just to follow common coding style can you add brackets? We usually don't use bracketless if statements.